documentation/md/style.md

```diff
@@ -619,6 +619,7 @@ Border:
  * **`text-border-width`** : The width of the border around the label.
  * **`text-border-style`** : The style of the border around the label; may be `solid`, `dotted`, `dashed`, or `double`.
  * **`text-border-color`** : The colour of the border around the label.
+ * **`text-border-radius`**: The radius of the border around the label.
```

Comment:
Will this work with all `text-background-shape`s, or will it only work with particular ones?

Reply:
if a text-border-radius is set, it will override the text-background-shape value

Reply:
That behaviour wouldn’t be consistent with existing style properties.